src/ci/resources/tes_application.conf

```diff
@@ -7,7 +7,17 @@ filesystems {
     global {
       class = "cromwell.filesystems.blob.BlobFileSystemConfig"
       config {
-        workspace-manager-url: "https://workspace.dsde-dev.broadinstitute.org"
+        container: "cromwell"
+        endpoint: "https://<storage-account>.blob.core.windows.net"
+        subscription: "00001111-2222-3333-aaaa-bbbbccccdddd"
+        # WSM config is needed for accessing WSM-managed blob containers
+        # created in Terra workspaces.
+        workspace-manager {
+          url: "https://workspace.dsde-dev.broadinstitute.org"
+          workspace-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+          container-resource-id: "00001111-2222-3333-aaaa-bbbbccccdddd"
+          overrideB2cToken: "Zardoz"
+        }
```

Comment:
I'm realizing I don't really understand the difference between global config and singleton config. This looks the same as how we were specifying global config before, but we're reading it as singleton config in `BlobPathBuilderFactory`.

Reply:
So prior I thought we were using the singletonConfig as well. When I have added config fields to the global field of the filesystem config it is processed as a separate singletonConfig. I tried to mess with this a little but had trouble getting anything to the global config object. I agree that it seems like this shouldn't be necessary, but the examples I've seen either do not have  configs, or define a singleton object.

Reply:
Following up - I understand these singletons better now and moved this config into a typed singleton config object.